scss/layout/_grid.scss

```diff
@@ -52,31 +65,29 @@
   }
 }
 
+// BREAKPOINT: MEDIUM
+@include media($med) {
```

Comment:
Ah! Well if the above media query handles `$med` - `$lg` then should this one just be `$lg` and up?